### PR TITLE
yaml support and chronos cli

### PR DESCRIPTION
**Pull request by navidurrahman**
Added support to create cronjobs using YAML files, you can specify all arguments in the yaml file, otherwise the job will be scheduled with default settings. In addition, environment files can be passed through -e argument

### CODE REVIEW

File: bin/chronos [hunk 1]
```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+import subprocess
+import yaml
+import os
+import json
+import argparse
+
+import chronos
+import texttable
```
Comment (solarkennedy) [inline bin/chronos:8]: I don't see this in any requirements.txt or setup.py

File: README.md [hunk 2]
```diff
@@ -47,3 +47,32 @@ c.delete_tasks("job123")
 * `chronos-nagios.py` - Nagios/Icinga style monitor of jobs
 `chronos-nagios.py --hostname chronos.server.com:4400 --crit 3 --prefix etl. --prefix data.`
 `chronos-nagios.py --hostname chronos.server.com:4400 --crit 3 --exclude etl.`
+
+* `chronos` - Nagios/Icinga style monitor of jobs
```
Comment (solarkennedy) [inline README.md]: This line needs to be updated

**Comment by solarkennedy** [comment]
I will leave it up to @asher regarding the new feature. This could easily grow into a project into itself.
Also there are no tests?

File: bin/chronos (hunk 1 above)
Comment (navidurrahman) [inline bin/chronos:8]: updated requirements list.

**Comment by navidurrahman** [comment]
@solarkennedy I wrote some test cases and it's not 100% completed. If @asher gives me a go ahead with this feature, I'll fix the test cases and push.

**Pull request closed without merging**